packages/jest-jasmine2/src/jasmine/jasmineLight.ts

```diff
@@ -40,10 +40,23 @@ import Timer from './Timer';
 import createSpy from './createSpy';
 import SpyRegistry from './spyRegistry';
 
+const testTimeoutSymbol = Symbol.for('TEST_TIMEOUT_SYMBOL');
+
 export const create = function (createOptions: Record<string, any>): Jasmine {
   const j$ = {...createOptions} as Jasmine;
 
-  j$._DEFAULT_TIMEOUT_INTERVAL = createOptions.testTimeout || 5000;
+  Object.defineProperty(j$, '_DEFAULT_TIMEOUT_INTERVAL', {
+    configurable: true,
+    enumerable: true,
+    get() {
+      return (
+        (global as any)[testTimeoutSymbol] || createOptions.testTimeout || 5000
+      );
+    },
+    set(value) {
+      (global as any)[testTimeoutSymbol] = value;
+    },
```

Comment:
Setter is used by `jasmine.DEFAULT_TIMEOUT_INTERVAL`. Otherwise it is unnecessary here.